### PR TITLE
First pass PGE 16.1 docs

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Added a PGE 16 section with 16.1 release notes.

### CODE REVIEW

File: product_docs/docs/pge/16/release_notes/index.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: "Release notes"
+navigation:
+ - rel_notes16.1
+---
+The EDB Postgres Extended Server documentation describes the latest version of EDB Postgres Extended Server 15,
```
Comment (josh-heyer) [inline product_docs/docs/pge/16/release_notes/index.mdx:6]: 16 😉

**Review by josh-heyer — CHANGES_REQUESTED** [review]
All of the install files should be generated (`npm run install-docs:build; npm run install-docs:deploy`) - to enable this, modify install_template/config.yaml to add `16` to the list of versions for each platform under EDB Postgres Extended Server

**Review by nidhibhammar — DISMISSED** [review]

**Comment by nidhibhammar** [comment]
Sorry, by mistake, I approved the changes.

File: product_docs/docs/pge/16/release_notes/index.mdx (hunk 1 above)
Comment (djw-m) [reply product_docs/docs/pge/16/release_notes/index.mdx]: Fixed

**Review by josh-heyer — APPROVED** [review]

**Pull request merged**